keps/sig-apps/1591-daemonset-surge/README.md

```diff
@@ -141,6 +145,28 @@ pod). If the old pod delays deletion, then the new pod has a chance to accept ha
   * Testing should set up conflicting rules like HostPort and verify that surge fails and the correct daemonset condition is set and events are generated.
 	* A test should cover a pod going unready during rollout and verifying it is immediately replaced.
 
+
+#### Prerequisite testing updates
+[x] I/we understand the owners of the involved components may require updates to existing tests to make this code solid enough prior to committing the changes necessary to implement this enhancement.
+
+##### Unit tests
+
+
+`k8s.io/kubernetes/pkg/apis/apps/validation`                     `06/06/2022`:     `90.6% of statements`
+`k8s.io/kubernetes/pkg/apis/apps/validation/validation.go:387`:	 `06/06/2022`:		`100.0% of statements`
+`k8s.io/kubernetes/pkg/controller/daemon`: `06/06/2022`: `70.7% of statements`
+`k8s.io/kubernetes/pkg/registry/apps/daemonset`: `06/06/2022`: `31.1% of statements`
+`k8s.io/kubernetes/pkg/registry/apps/daemonset/strategy.go:129`:	`06/06/2022`:	`100.0% of statements`
+
+##### Integration tests
+
+We've enough coverage in the unit and e2e tests. No new integration tests are added
+
+##### e2e tests
+
+An e2e test which exercises maxSurge when surge is used as update strategy is added for daemonsets.
```

Comment:
Is this new or existing, also from the template:
```
For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
https://storage.googleapis.com/k8s-triage/index.html
```

Reply:
We added it in the alpha stage itself. I've updated the test link